source/base/Text.ooc

```diff
@@ -149,6 +149,30 @@ Text: cover {
 			--rightPosition
 		this slice(leftPosition, rightPosition - leftPosition + 1)
 	}
+	replaceAll: func (toReplace, newValue: This) -> This {
+		t := this take()
+		result: This
+		position := t find(toReplace)
+		if (position == -1)
+			result = t copy()
+		else {
+			previous := 0
+			result = This new()
+			x := toReplace take()
+			y := newValue take()
```

Comment:
Yeah, I can accept `t`, but `x` and `y` are pretty non-explanatory. `toReplaceTaken` and `newValueTaken`, is that too much?